src/components/z-book-card/styles.css

```diff
@@ -248,7 +248,7 @@
 
 :host > article.search .wrapper-container .action-container {
   padding-bottom: calc(var(--space-unit) * 2);
-  margin: auto;
+  margin: auto 0 0;
```

Comment:
Non è sufficiente fare `margin: 0 auto;`? In questo modo mantieni la centratura del div (ora non c'è più) e il margin-bottom è 0 come vuoi tu.

Reply:
in realtà il nostro scopo è avere: 
margin-left: 0;
margin-right: 0;
margin-bottom: 0;
margin-top: auto (più per retrocompatibilità che per bisogno specifico della storia)

quindi o mettiamo `margin: auto 0 0` o se non ci interessa il margin-top: auto possiamo mettere `margin:0`

Reply:
Perché margin left e right a 0 e non auto? cosa cambia? Almeno il contenuto si centra.. 
Tanto lato dashboard bisogna comunque mettere un div largo 100% con dentro il suo stile, no?